<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>20</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="lib1_des_text">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>381</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Fangsong Std</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="lib1_src_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>381</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Fangsong Std</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="lib1_selbt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lib1_svbt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lib1_exebt">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>词法分析</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>词法分析</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
